astroquery/astrometry_net/core.py

```diff
@@ -0,0 +1,386 @@
+"""
```

Comment:
these docs might belong in `docs/astrometry_net/[something].rst`

Reply:
Agreed; this is a holdover from the older PR.